x-pack/plugins/index_lifecycle_management/public/index.scss

```diff
@@ -1,3 +1,6 @@
+// Import the EUI global scope so we can use EUI constants
+@import 'src/legacy/ui/public/styles/_styling_constants';
+
 .policyTable__horizontalScrollContainer {
```

Comment:
A couple things:

1. Selectors should not be directly in `index` files. You'll need another file that the index imports.
2. You don't need the styling constants imports when importing the SASS in the plugin .ts file. That's automatically added.

Reply:
Thanks @cchaos! Changes are in-flight to address your comments.

> Selectors should not be directly in index files. You'll need another file that the index imports.

Could you explain the problem this solves?

Reply:
Yes, it solves the problem of scaleability of the styles. If more UI components and files are added to this plugin, each would get it's own SASS file and get imported via the index file. Then if any are also removed, it's easy to remove the entire folder and a single import line vs having to comb through a list of selectors to find which ones should be removed (or risk keeping dead code).

Reply:
Thanks for the context! That is very helpful.